### ISSUE TITLE
"pong" reply option

### ISSUE THREAD

**Issue by RollingHog** [discord-buttons/discord-buttons#154]
As Discord doc states, there is a "pong" option for Buttons API, which means "click aknowledged, don't do anything". I can't see it here either in doc/examples or in code. Is it present in this lib? If no, could someone please add it? It would be really useful thing to have.

**Comment by AngeloCore** [comment]
ok

**Comment by xHyroM** [comment]
> As Discord doc states, there is a "pong" option for Buttons API, which means "click aknowledged, don't do anything". I can't see it here either in doc/examples or in code. Is it present in this lib? If no, could someone please add it? It would be really useful thing to have.

`button.defer()` ?

**Comment by RollingHog** [comment]
> > As Discord doc states, there is a "pong" option for Buttons API, which means "click aknowledged, don't do anything". I can't see it here either in doc/examples or in code. Is it present in this lib? If no, could someone please add it? It would be really useful thing to have.
> 
> `button.defer()` ?

Thanks for responce. 

No, defer is "thinking, will respond later". It shows button as "loading"
https://discord.com/developers/docs/interactions/slash-commands#interaction-response-object-interaction-callback-type

**Comment by xHyroM** [comment]
> > > As Discord doc states, there is a "pong" option for Buttons API, which means "click aknowledged, don't do anything". I can't see it here either in doc/examples or in code. Is it present in this lib? If no, could someone please add it? It would be really useful thing to have.
> > 
> > 
> > `button.defer()` ?
> 
> Thanks for responce.
> 
> No, defer is "thinking, will respond later". It shows button as "loading"
> https://discord.com/developers/docs/interactions/slash-commands#interaction-response-object-interaction-callback-type

Oh, okay.

**Comment by Erfesto** [comment]
If you just need kill stupid "interaction error", you can use
`button.reply.defer()`
It not gives any errors in console, or in discord.

For example,
`bot.on('clickButton', async (button) => {
      if(button.clicker.user.id == myid) {
        if(button.id === 'my_button') {
          console.log('Clicked, yay!')
          button.reply.defer()
        }
      }
});

I believe, discord devs will "fix" it, and then we will use "reply" and then "message.delete()".
They already killed slashcommands with that (first versions of slapshcommads has ~7 types, now only ~2).

**Comment by RollingHog** [comment]
Could not answer/reproduce for a while due to discord.js v13 *awesome backwards compatability.*
First, I literally said "no, defer is not what I want" one message above. Have you read it, dear sir?
Second… Suddenly I tested it and saw that after about half a second button goes from "loading" back to "normal" state. This is still not fully what I want, but it is acceptable. 
**It is important thing which should be added to your documentation to avoid such questions in the future.** How can I send a PR/add it to your wiki?

**Comment by Erfesto** [comment]
https://github.com/discord-buttons/docs/blob/main/docs/docs/stable/examples/reply.md

**Comment by AngeloCore** [comment]
`defer()` is type `6` so send a reply with nothing
`think()` is type `5` so send a reply with "**Test Bot** is thinking..."

But if you want to use type `1`, you can use `reply.send('content', { type: 1 });`

**Comment by RollingHog** [comment]
> `defer()` is type `6` so send a reply with nothing
> `think()` is type `5` so send a reply with "**Test Bot** is thinking..."
> 
> But if you want to use type `1`, you can use `reply.send('content', { type: 1 });`

YES! The exact thing I was looking for. Could you please add note about this option in "reply" doc?

**Comment by AngeloCore** [comment]
I'm sorry about this time we will do last update of discord-buttons, I know documentation is not finished but I just don't have much time because I'm working on my better discord library

**Comment by AngeloCore** [comment]
Whoever wants, can feel free to do it

**Comment by RollingHog** [comment]
I can do this, it's not this much, but where can I edit doc?

**Comment by RollingHog** [comment]
Done a pull request with proposed changes: #19

**Comment by RollingHog** [comment]
Err, wrong link. Here: https://github.com/discord-buttons/docs/pull/19

**Comment by Dragonizedpizza** [comment]
> I'm sorry about this time we will do last update of discord-buttons, I know documentation is not finished but I just don't have much time because I'm working on my better discord library

Sneak peak?
